### ISSUE TITLE
How to Run Instructions

### ISSUE THREAD

**Issue by ctrlbee** [hugbug/conpianist#80]
Could you please add instructions on how to run this application on a Mac to the readme? Tried g++ -o yamaha Main/Main.cpp but it's missing some dependencies. Thanks.

**Comment by hugbug** [comment]
The ready to run Mac app can be downloaded from [releases section](https://github.com/hugbug/conpianist/releases).

**Comment by ctrlbee** [comment]
Awesome, thanks. Got it, but having OS compatibility issues: "Check with the developer to make sure ConPianist works with this version of macOS. You may need to reinstall the application. Be sure to install any available updates for the application and macOS." I'm currently on Catalina (but planned to update to Big Sur soon)

**Comment by hugbug** [comment]
Indeed. There was an issue with the Mac build. The fixed version isn't released yet. I can send you current development version, which should work. Please send me a message to hugbug@users.sourceforge.net.
